crates/mockiato-codegen/src/syn_ext.rs

```diff
@@ -0,0 +1,15 @@
+use syn::{Ident, Path};
+
+pub(crate) trait PathExt {
+    fn first_segment_as_ident(&self) -> Option<&Ident>;
+}
+
+impl PathExt for Path {
+    fn first_segment_as_ident(&self) -> Option<&Ident> {
+        if self.segments.is_empty() {
```

Comment:
How about `Some(&self.segments.get(0)?.ident)`?

Reply:
Addressed in 35e3e81